src/pwr.ps1

```diff
@@ -1,1426 +1,265 @@
-			Write-PwrFatal 'prune not compatible with -Packages'
-		}
-		Lock-PwrLock
-		try {
-			$LoadCache = Get-PwrLoadCache
-			PrunePwrPackages $LoadCache
-			Save-PwrLoadCache $LoadCache
-		} finally {
-			Unlock-PwrLock
-		}
-	}
-	{$_ -in 'rm', 'remove'} {
-		Lock-PwrLock
-		try {
-			if ($DaysOld) {
-				Write-PwrWarning "-DaysOld is deprecated in favor of 'prune' and will be removed in a future version"
-				if ($Packages.Count -gt 0) {
-					Write-PwrFatal '-DaysOld not compatible with -Packages'
-				}
-				$Pkgs = Get-InstalledPwrPackages
-				foreach ($Name in $Pkgs.Keys) {
-					foreach ($Ver in $Pkgs.$Name) {
-						$PkgRoot = "$PwrPkgPath\$Name-$Ver"
-						try {
-							$Item = Get-ChildItem -Path "$PkgRoot\.pwr"
-						} catch {
-							Write-PwrWarning "$PkgRoot is not a pwr package; it should be removed manually"
-							continue
-						}
-						$Item.LastAccessTime = $Item.LastAccessTime
-						$Old = $Item.LastAccessTime -lt ((Get-Date) - (New-TimeSpan -Days $DaysOld))
-						if ($Old -and $PSCmdlet.ShouldProcess("${Name}:$Ver", 'remove pwr package')) {
-							Write-PwrOutput "removing ${Name}:$Ver ... " -NoNewline
-							Remove-Directory $PkgRoot
-							Write-PwrHost 'done'
-						}
-					}
-				}
-			} else {
-				Assert-NonEmptyPwrPackages
-				foreach ($P in $Packages) {
-					$Pkg = Assert-PwrPackage $P
-					if ($Pkg.Local) {
-						Write-PwrFatal "tried to remove local package $($Pkg.Ref)"
-					} elseif ("$env:PwrLoadedPackages".Contains($Pkg.Ref)) {
-						Write-PwrFatal "tried to remove loaded package $($Pkg.Ref)"
-					} elseif ("$env:PwrShellPackages".Contains($Pkg.Ref)) {
-						Write-PwrFatal "tried to remove shell session package $($Pkg.Ref)"
-					} elseif (Test-PwrPackage $Pkg) {
-						if ($PSCmdlet.ShouldProcess($Pkg.Ref, 'remove pwr package')) {
-							Write-PwrOutput "removing $($Pkg.Ref) ... " -NoNewline
-							$Path = Resolve-PwrPackage $Pkg
-							Remove-Directory $Path
-							Write-PwrHost 'done'
-						}
-					} else {
-						Write-PwrFatal "$($Pkg.Ref) not found"
-					}
-				}
-			}
-		} finally {
-			Unlock-PwrLock
-		}
-	}
-	'which' {
-		Assert-NonEmptyPwrPackages
-		foreach ($local:P in $Packages) {
-			$local:Pkg = Assert-PwrPackage $local:P
-			Write-PwrHost "$($local:Pkg.Ref) $($local:Pkg.Digest)"
+Set-Alias -Name 'airpower' -Value 'Invoke-Airpower' -Scope Global
+Set-Alias -Name 'air' -Value 'Invoke-Airpower' -Scope Global
+Set-Alias -Name 'pwr' -Value 'Invoke-Airpower' -Scope Global
+
+& {
+	if ('Airpower.psm1' -eq (Split-Path $MyInvocation.ScriptName -Leaf)) {
+		# Invoked as a module
+		$params = @{
+			URL = "https://www.powershellgallery.com/packages/airpower"
+			Method = 'HEAD'
 		}
-	}
-	'where' {
-		Assert-NonEmptyPwrPackages
-		foreach ($local:P in $Packages) {
-			$local:Pkg = Assert-PwrPackage $local:P
-			if (-not (Test-PwrPackage $local:Pkg)) {
-				Write-PwrFatal "package $($local:Pkg.Ref) is not installed"
+		$resp = HttpRequest @params | HttpSend -NoRedirect
```

Comment:
When loading `Airpower` in an offline environment (starting powershell or `Import-Module Airpower`), this line causes the following error:
```pwsh
Exception calling "GetResult" with "0" argument(s): "An error occurred while sending the request."
At ***\Airpower.psm1:38 char:2
+     $resp = $cli.SendAsync($Req, [Net.Http.HttpCompletionOption]::Res ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : HttpRequestException
```

Reply:
Fixed